src/common/prop.c

```diff
@@ -705,6 +738,9 @@ daos_prop_copy(daos_prop_t *prop_req, daos_prop_t *prop_reply)
 		if (roots_alloc)
 			free_ptr_prop_entry(prop_req, DAOS_PROP_CO_ROOTS);
 
+		if(policy_alloc)
```

Comment:
(style)  space required before the open parenthesis '('

Reply:
please address.